components/mpas-seaice/bld/build-namelist-section

```diff
@@ -28,7 +28,7 @@ add_default($nl, 'config_full_abort_write');
 # Namelist group: decomposition #
 #################################
 
-add_default($nl, 'config_block_decomp_file_prefix', 'val'=>"'${DIN_LOC_ROOT}/ice/mpas-cice/${ICE_GRID}/mpas-cice.graph.info.${date_stamp}.part.'");
+add_default($nl, 'config_block_decomp_file_prefix', 'val'=>"'${DIN_LOC_ROOT}/ice/mpas-seaice/${ICE_GRID}/mpas-seaice.graph.info.${date_stamp}.part.'");
```

Comment:
@akturner, @jonbob do either of you know where this might get used and what the consequences of renaming it might be.  Note that the corresponding line in `build-namelist` does not have `mpas-seaice.graph.info.` hard coded like here.